pkg/graveler/errors.go

```diff
@@ -59,7 +59,7 @@ var (
 	ErrTooManyTries                 = errors.New("too many tries")
 	ErrSkipValueUpdate              = errors.New("skip value update")
 	ErrImport                       = wrapError(ErrUserVisible, "import error")
-	ErrReadOnlyRepository           = errors.New("read-only repository")
+	ErrReadOnlyRepository           = wrapError(ErrUserVisible, "read-only repository")
```

Comment:
Sorry for bugging again on this: maybe even add a suggestion?
```suggestion
	ErrReadOnlyRepository           = wrapError(ErrUserVisible, "read-only repository, will need to use force")
```